swap_meet/clothing.py

```diff
@@ -1,2 +1,17 @@
-class Clothing:
-    pass
\ No newline at end of file
+import uuid
+from swap_meet.item import Item
+class Clothing(Item):
+
+    def __init__(self, id=None, condition=None, fabric=None) :
+        super().__init__(id, condition)
+        if fabric:
+            if type(fabric) is not str:
```

Comment:
Nice extra error checking!

I would do this before the other code, so that the error is generated first, before extra work is unnecessarily done.